misc/python/materialize/xcompile.py

```diff
@@ -158,8 +158,13 @@ def cargo(
 
     if sys.platform == "darwin":
         _bootstrap_darwin(arch)
+        homebrew_prefix = spawn.capture(["brew", "--prefix"])
         sysroot = spawn.capture([f"{_target}-cc", "-print-sysroot"]).strip()
-        rustflags += [f"-L{sysroot}/lib"]
+        rustflags += [
+            f"-L{sysroot}/lib",
+            "-Clink-arg=-fuse-ld=lld",
+            f"-Clink-arg=-B{homebrew_prefix}/bin",
+        ]
```

Comment:
Including the whole Homebrew prefix here is a bit risky, because we don't want to use `cc1` or other tools that might be in `/opt/homebrew/bin`. We just want to use `lld`. Something like this should work though!

```
        lld_prefix = spawn.capture(["brew", "--prefix", "lld"])
        rustflags += [..., "-Clink-arg=-fuse-ld=lld", f"-Clink-arg=-B{lld_prefix}/bin"]
```

Reply:
True, done.